HACKING.md

```diff
@@ -0,0 +1,89 @@
+# Hacking on Quorum / various notes
+
+## Testing with Constellation
+
+### `tm.conf`
+
+Replace with appropriate absolute paths:
+
+TODO(joel): figure out how to use relative paths
```

Comment:
Relative path should work just fine for Constellation config and PRIVATE_CONFIG env var. What's the issue?